frontend/src/components/summary/group/CCFRISummary.vue

```diff
@@ -429,8 +429,10 @@ import _ from 'lodash';
 import { isChangeRequest } from '../../../utils/common.js';
```

Comment:
could you please replace all "../../.." with "@" for all files in your PR? Thank you!

Reply:
Changes included in PR#536